test/block_test.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'test_helper'
+
+describe 'Block' do
+  before do
+    d_r = Hotel::DateRange.new Date.today, Date.today.next
```

Comment:
Inlining this variable would have been better than giving it such a non-descriptive name.  Generally we reserve very short variable names for things like loop variables.